src/api/auth/src/index.js

```diff
@@ -1,6 +1,7 @@
-const { Satellite } = require('@senecacdot/satellite');
+const { Satellite, Redis } = require('@senecacdot/satellite');
 const passport = require('passport');
 const session = require('express-session');
+const RedisStore = require('connect-redis')(session);
```

Comment:
Reading what `express-session` is, I assume we just pass in a new cookie with no options?